x-pack/plugins/lens/public/drag_drop/drag_drop.tsx

```diff
@@ -299,6 +335,19 @@ const DragInner = memo(function DragDropInner({
                 }
               } else if (e.key === keys.ESCAPE) {
                 dragEnd();
+              } else if (
+                keyboardMode &&
+                order &&
+                (keys.ARROW_LEFT === e.key ||
+                  keys.ARROW_RIGHT === e.key ||
+                  'd' === e.key ||
```

Comment:
I temporarily added WSAD keys as arrows don't work for me correctly for arrows :(